_includes/repository/repo.liquid

```diff
@@ -5,6 +5,26 @@
   {% assign show_owner = true %}
 {% endif %}
 
+{% assign lang = site.active_lang | split: '-' | first %}
+
+{% case lang %}
+  {% when 'pt' %}
+    {% assign lang = site.active_lang %}
+
+  {% when 'zh' %}
+    {% assign lang_last = site.active_lang | split: '-' | last %}
+    {% case lang_last %}
+      {% when 'cn', 'sg', 'my', 'hans' %}
+        {% assign lang = 'cn' %}
+      {% when 'tw', 'hk', 'mo', 'hant' %}
+        {% assign lang = 'zh-tw' %}
+
+        {% comment %} when... add a new language here if needed {% endcomment %}
```

Comment:
This comment must be after the `endcase`, if not the language would be added inside the `zh` case.